Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM ubuntu:22.04
 
-LABEL org.label-schema.maintainer="betadots GmbH <info@betadots.de>"
+LABEL org.label-schema.maintainer="betadots GmbH <info@betadots.de>" \
+      org.label-schema.vendor="betadots" \
+      org.label-schema.url="https://github.com/betadots/pdc" \
+      org.label-schema.name="Puppet Development Container" \
+      org.label-schema.license="Apache-2.0" \
```

Comment:
we dont have a licence yet, and dont we take agpl like in hdm?

Reply:
I had to do a bit of research about the scope first. This label is only, to my understanding, about our own software in the repo. Which is basically only the Dockerfile/Gemfile. I updated it to AGPL3 as well.